backend/src/prisma/schema.prisma

```diff
@@ -83,3 +84,12 @@ model Rating {
   createdAt       DateTime @default(now())
   anonymous       Boolean  @default(false)
 }
+
+model UserSocialAccount {
+  id              String   @id @default(cuid())  
+  provider        String   @unique                  
```

Comment:
Maybe you can add another enum named `Provider` in prisma and tap that into this field. it will help the code in the longer run!